armeria/pom.xml

```diff
@@ -39,35 +39,35 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>2.0.62.Final</version>
+      <version>2.0.63.Final</version>
```

Comment:
IIUC next netty release, and subsequent armeria release, we'll be able to remove this

Reply:
looking forward to it!

Reply:
added the missing comment